hunts/src/api.js

```diff
@@ -3,9 +3,27 @@ import Cookies from "js-cookie";
 function getPuzzles(huntId) {
   const puzzlesApiUrl = `/api/v1/hunt/${huntId}/puzzles`;
   return fetch(puzzlesApiUrl).then((response) => {
-    if (response.status > 400) {
+    if (response.status >= 400) {
       // TODO: error handling
-      console.error("Get puzzles API failure", response);
+      throw new Error("Get puzzles API failure");
+    }
+    return response.json();
+  });
+}
+
+function addPuzzle(huntId, data) {
+  const puzzlesApiUrl = `/api/v1/hunt/${huntId}/puzzles`;
+  return fetch(puzzlesApiUrl, {
+    method: "POST",
+    headers: {
+      "X-CSRFToken": Cookies.get("csrftoken"),
+      "Content-Type": "application/json",
```

Comment:
Seems like we use the same headers for various requests throughout these React forms. Is there a best practice to avoid repeating these? Or is it more conventional to explicitly repeat them?